_faq/writing-a-great-proposal.md

```diff
@@ -25,4 +25,4 @@ Please follow the **application template** in the faq. All fields are required i
 
 For writing your proposal we recommend using [Google Docs](https://www.google.com/docs/about/) as this will be supported by the Google submission system.
 
-If needed, you can get further information about GSoC [here](http://write.flossmanuals.net/gsocstudentguide/what-is-google-summer-of-code/).
+If needed, you can get further information about GSoC [here](https://google.github.io/gsocguides/student/).
```

Comment:
Why this link instead of the GSoC site?

Reply:
Since people who are going to see this are going to be gsoc applicants so i thought this site would be better as the orginal gsoc site  also has the parts for the organisation to take part in gsoc. Also in my personal experience this site is more detailed than https://summerofcode.withgoogle.com/ for newcomers that's why i have this link i thought this was also maintined by Google anyway right??
Shall i change the link then..

Reply:
@abhishalya it's been almost a month, should I change the link ??

Reply:
Yes, change that. Idk why they have a separate site. But they might remove it in long run. I'd rather have the main site here.